service_container/debug.rst

```diff
@@ -6,17 +6,22 @@ How to Debug the Service Container & List Services
 ==================================================
 
 You can find out what services are registered with the container using the
-console. To show all services and the class for each service, run:
+console. To show all services (public and private) and their PHP classes, run:
 
 .. code-block:: terminal
 
     $ php bin/console debug:container
 
-By default, only public services are shown, but you can also view private services:
+Hidden services, those whose ID starts with a dot character (``.``) aren't
+included by default in the output of this command. Add the ``--show-hidden``
+option to list them too:
```

Comment:
Do we really need the paragraph? In other examples, we use inline comments which IMO would fit here well too:

```rst
.. code-block:: terminal

    $ php bin/console debug:container

    # use the --show-hidden option to display hidden services (services whose ID starts with a dot)
    $ php bin/console debug:container --show-hidden
```